scheduler/src/cook/mesos/api.clj

```diff
@@ -424,11 +473,14 @@
                                 :host-placement
                                 make-host-placement-txn)
         group-txn {:db/id (d/tempid :db.part/user)
-                    :group/uuid uuid
-                    :group/name (:name group)
-                    :group/host-placement hp-txn-id
-                    :group/job job-db-ids}]
-      (conj hp-txns group-txn)))
+                   :group/uuid uuid
+                   :group/name (:name group)
+                   :group/host-placement hp-txn-id
+                   :group/job job-db-ids
+                   :group/straggler-handling (-> group
+                                                 :straggler-handling
+                                                 (make-straggler-handling-txn))}]
```

Comment:
Ah, interesting. I just read about components, seem like HostPlacement should be component entities. Should this change be added to this PR?